samples/projects/cdktf-typescript/package.json

```diff
@@ -41,5 +39,8 @@
     "ts-jest": "^29.0.3",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.3"
+  },
+  "dependencies": {
+    "@openfabr/package-ri-cdktf": "file:../../packages/cdktf-typescript"
```

Comment:
arg can't reference the publish package properly until published. 

@cwang any idea of a better workflow here?

Reply:
Maybe just publish that package first?

Reply:
Merge as is to publish, then another PR to update the ref to the published package?

Reply:
Ideally you want to split this PR into two parts and it's always package first and project later, according to how it's currently been set up. Otherwise this PR would generate a broken release.

Reply:
@cwang I think we need to merge this (and then #72) as is and then patch this package dependency. Agree ideally should split but a this point I think it's not work the time. Not having these changes published is also holding back other work now like testing runner on prod mode and on the staging server.

Package code is tested and good to go.

Reply:
Let me rearrange the branch a little to merge it.